Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt

```diff
@@ -49,7 +49,7 @@ class BpkThemeTest {
 
     val newContext = BpkTheme.wrapContextWithDefaults(activity)
     newContext.theme.resolveAttribute(R.attr.bpkPrimaryColor, t, true)
-    Assert.assertEquals(blue500, t.data)
+    Assert.assertEquals(skyBlue, t.data)
```

Comment:
Calendar theme attributes (calendarDateSelectedBackgroundColor) etc need to be updated as well.

Reply:
done